ssv/sync_committee_aggregator.go

```diff
@@ -120,14 +123,9 @@ func (r *SyncCommitteeAggregatorRunner) ProcessPreConsensus(signedMsg *types.Sig
 		DataSSZ: byts,
 	}
 
-	if anyIsAggregator {
-		if err := r.BaseRunner.decide(r, input); err != nil {
-			return errors.Wrap(err, "can't start new duty runner instance for duty")
-		}
-	} else {
-		r.BaseRunner.State.Finished = true
```

Comment:
this is the fix right?
So we don't mark it as finished too early?

Reply:
Fix is to not call `GetSyncCommitteeContribution` when you have no aggregator duties.

So the "if no aggregators, set Finished to true" is moved up before that call.